src/industry_cmd.cpp

```diff
@@ -2717,6 +2744,10 @@ static void ChangeIndustryProduction(Industry *i, bool monthly)
 		}
 	}
 
+	/* If override flags are set, prevent actually changing production if any was decided on */
```

Comment:
The code above already triggers news reports about the change.
So altering the results here results in lying news.